api/services/gameService.js

```diff
@@ -111,7 +120,8 @@ module.exports = {
                   if (game.players.length > 1) {
                     const p0 = userService.findUser({ userId: game.players[0].id });
                     const p1 = userService.findUser({ userId: game.players[1].id });
-                    return Promise.all([Promise.resolve(game), p0, p1]);
+                    const spectators = fetchSpectatorUsernames(game.id);
```

Comment:
I'm having an issue when this function is called, I am getting an empty array and it seems like it's just skipping this function completely.